tests/rest/test_connection.py

```diff
@@ -2443,6 +2443,16 @@ def test_basic(self, con120):
             }
         }
 
+    def test_basic_connectionless(self, con120):
+        cube = openeo.DataCube.load_stac("https://provider.test/dataset")
+        assert cube.flat_graph() == {
+            "loadstac1": {
+                "process_id": "load_stac",
+                "arguments": {"url": "https://provider.test/dataset"},
+                "result": True,
+            }
+        }
+
```

Comment:
This does not really belong in test_connection.py as this test is about _not_ using a connection 

I would put it in
https://github.com/Open-EO/openeo-python-client/blob/master/tests/rest/datacube/test_datacube.py

Reply:
Moved. And made it more explicit that the processgraph should be the same as the version with connection